src/__snapshots__/index.test.js.snap

```diff
@@ -576,6 +580,33 @@ public enum Duration {
     public static let duration6: TimeInterval = 0.35
 }
 
+public enum FontWeight {
+    case normal
+    case bold
+}
+
+public enum Font {
+    public static let title1Size: CGFloat = 28
+    public static let title1Weight: FontWeight = .bold
+    public static let title2Size: CGFloat = 28
+    public static let title2Weight: FontWeight = .bold
+    public static let title3Size: CGFloat = 22
+    public static let title3Weight: FontWeight = .bold
+    public static let title4Size: CGFloat = 20
+    public static let title4Weight: FontWeight = .bold
+    public static let title5Size: CGFloat = 18
+    public static let title5Weight: FontWeight = .bold
+    public static let title6Size: CGFloat = 16
+    public static let title6Weight: FontWeight = .bold
+    public static let title7Size: CGFloat = 14
+    public static let title7Weight: FontWeight = .bold
+    public static let title8Size: CGFloat = 12
+    public static let title8Weight: FontWeight = .bold
+    public static let body1Size: CGFloat = 16
+    public static let body2Size: CGFloat = 14
+    public static let body3Size: CGFloat = 12
+}
```

Comment:
Would this also work if `title1Weight: FontWeight = .bold` were written as `public static let title1Weight: UIFont.Weight = bold` (or something along those lines)? 

I ask because I try to avoid introducing schema changes to the tokens if the functionality is only needed by one platform. (On Android, for example, the proper convention is to separate space, color, and other types into their own files. We put it all in one file regardless because there aren't large downsides.)

Reply:
iOS itself doesn't have a concept of font weights already, it's just a matter of us including two separate fonts and deciding which one to use for a given label.

Currently as this PR is set up, it's just generating constants for the font size and weight, and then in the iOS repo we're going to have to still map those to actual UIFont instances. However we could instead move the font assets themselves to be included here in ThumbprintTokens pod, and then generate code that instantiates the UIFont constants themselves in the format `public static let title1 = UIFontMetrics(forTextStyle: .headline).scaledFont(for: UIFont(name: "MarkForThumbtack-Bold", size: 28)!, compatibleWith: nil)`. However that would also require changing the schema in some way because currently `Font` and `Font Weight` are independent tokens in `tokens.js` but we'd be combining them into a single set of generated constants. In other words, it'd probably require a schema like
```
{
    name: 'Font',
    tokens: [
        id: 'title1',
        platforms: {
            scss: {
                name: '$tp-font_title_1',
                size: '28px',
                weight': '700',
                ...
            },
            ios: {
                name: 'title1',
                size: 'CGFloat(28)',
                weight: 'Bold',
                ...
            },
        },
    ],
}
```
and then separate handling in the handlebars template for interpreting that schema.

Reply:
Another option would be to leave the schema unchanged and just do `public static let title1Weight = "bold"`, but that would require us on the iOS side to then check
```
if title1Weight == "bold" {
    // Use "MarkForThumbtack-Bold"
} else {
    // Use "MarkForThumbtack-Regular"
}
```
which seems fragile because we're just comparing hardcoded strings

Reply:
Synced over slack and updated PR accordingly